packages/plasma-new-hope/src/components/Dropdown/Dropdown.types.ts

```diff
@@ -35,17 +35,22 @@ export type DropdownProps<T extends DropdownItemOption = DropdownItemOption> = {
      */
     onItemSelect?: (item: T, event: SyntheticEvent) => void;
     /**
-     * Способ открытия дропдауна окна - наведение или клик мышью.
+     * Способ открытия Dropdown окна - наведение или клик мышью.
      * @default click
      */
     trigger?: DropdownTrigger;
     /**
-     * Сторона открытия дропдауна относительно target элемента.
+     * Открывает дропдаун окно по правому клику мышью
```

Comment:
Тут тогда тоже должно быть так:

```suggestion
     * Открывает Dropdown окно по правому клику мышью
```